modules/search-pipeline-common/src/main/java/org/opensearch/search/pipeline/common/FilterQueryRequestProcessor.java

```diff
@@ -53,12 +54,13 @@ public String getType() {
     /**
      * Constructor that takes a filter query.
      *
-     * @param tag         processor tag
-     * @param description processor description
+     * @param tag            processor tag
+     * @param description    processor description
+     * @param ignoreFailure  option to ignore failure
      * @param filterQuery the query that will be added as a filter to incoming queries
      */
-    public FilterQueryRequestProcessor(String tag, String description, QueryBuilder filterQuery) {
-        super(tag, description);
+    public FilterQueryRequestProcessor(String tag, String description, boolean ignoreFailure, QueryBuilder filterQuery) {
```

Comment:
I hadn't noticed before, but this constructor probably shouldn't be public, since it should only get called from the `create` method in the factory. 

It's not important for this PR (and not very important in general), but eventually we should probably change it.

Reply:
It was like that for some time, but I can fix this by the way

Reply:
made the new commit to fix it [fix filter query and change log warn message](https://github.com/opensearch-project/OpenSearch/pull/8373/commits/10344528c6caebdb00bc0fb15b0b0df29136141d)